src/aap_eda/api/serializers/project.py

```diff
@@ -247,3 +271,27 @@ class Meta:
         model = models.ExtraVar
         fields = ["id"]
         read_only_fields = ["id"]
+
+
+def get_proxy_for_display(proxy: str) -> str:
+    if not (proxy.startswith("http://") or proxy.startswith("https://")):
+        proxy = f"http://{proxy}"
+    result = urlparse(proxy)
+    if "@" not in proxy:
+        return proxy
+    cred, domain = result.netloc.split("@")
+    if ":" in cred:
+        user, _ = cred.split(":")
+        domain = f"user:{ENCRYPTED_STRING}@{domain}"
+    else:
+        domain = f"{ENCRYPTED_STRING}@{domain}"
```

Comment:
If you are going to hide whichever is before the "@" you don't need to search for an user in line 284

Reply:
We still want to show {user} if it exists.